PocketKit/Tests/PocketKitTests/HomeViewModelTests.swift

```diff
@@ -688,7 +690,7 @@ class HomeViewModelTests: XCTestCase {
             )
         }
 
-        wait(for: [urlExpectation], timeout: 1)
+        wait(for: [urlExpectation], timeout: 5)
```

Comment:
curious to understand why the timeouts need to be 5

Reply:
@cyndichin  I think the tests were failing with '1' and tried with 5 but were still failing since the issue was different but I probably forget to change it back to 1, thanks for catching this.